src/cpp/rtps/writer/StatelessWriter.cpp

```diff
@@ -69,8 +69,13 @@ StatelessWriter::StatelessWriter(
 
 StatelessWriter::~StatelessWriter()
 {
```

Comment:
We don't need to disable the flow controller here as in `StatefulWriter`s destructor?

Reply:
I've forgotten it. Fixed.